API.md

```diff
@@ -102,18 +105,43 @@ toki.on('error', (error)=>{
 
 #### config.changed
 
-__toki__ subscribes to events triggered by __toki-config__ if the underlaying configuration mechanism detects a configuration change. __toki__ will bubble up the event for the __toki__ instantiatior to act accordingly in response to the event. 
+__toki__ subscribes to events triggered by __toki-config__ if the underlaying configuration mechanism detects a configuration change. __toki__ will bubble up the event for the __toki__ instantiatior to act accordingly in response to the event.
 
 ```Javascript
-//intantiate toki
+//instantiate toki
 const Toki = require('toki');
 const toki = new Toki({
     router : routerInstance
 });
 
 //wait for toki to be ready
 toki.on('config.changed', ()=>{
-    //toki is open for buisness (aka do your thing)    
+    //toki is open for business (aka do your thing)    
 });
 
 ```
+
+### Contracts
+_Fulfilled by the Bridge_
+
+These contracts supply a minimum. Additional properties or methods may exist, but should not be used since requests and responses provided by the bridge are often decorated versions of underlying requests or responses.
+
+#### Request
+
+Request is a decorated version of the Node http-server request object. It will always have the following:
+
++ `request.query` - a parsed query object
++ `request.params` - an object of any params from passed paths
++ `request.path` - the current path
++ `request.method` - the method which called this request
++ `request.headers` - an object containing all headers
+
+
+#### Response
```

Comment:
@dhinklexo does calling something like `response.send(payload)` from within the mdthod (like toki-method-http) simply attach that payload to the response that will eventually be sent by Toki? Or does it actually fire off the response object back to the client?

Basically do the toki plugins act like hapi prerequest handlers/decorators, or like the actual request handlers?